package.json

```diff
@@ -5,16 +5,20 @@
     "packages/*"
   ],
   "scripts": {
+    "build": "nx run-many --target=build",
     "checkNpmVersion": "./scripts/check-npm-version.sh",
     "clean": "./config/cli/clean-root.sh",
     "e2e:inject": "node ./scripts/e2e-inject-resolutions.js",
     "e2e:publish": "./scripts/e2e-publish.sh",
     "e2e:resolutions": "node ./scripts/e2e-resolutions.js",
     "examples": "npm run examples --workspaces --if-present",
+    "lint": "nx run-many --target=lint",
     "preinstall": "npm run checkNpmVersion",
     "postinstall": "npm run build --workspaces",
     "prepare": "git config --local core.hooksPath .githooks",
     "prettier": "prettier --write \"./**/*.{js,json,md,ts,yml}\"",
+    "test": "nx run-many --target=test",
```

Comment:
Since I configured test to depend on build it will build any dependent packages and also run tests in parallel. By default it does 3 threads